Header header

# курс
float64 heading

# крен
float64 roll

# дифферент
float64 pitch